flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/join/LookupJoinTest.scala

```diff
@@ -951,6 +951,58 @@ class LookupJoinTest(legacyTableSource: Boolean) extends TableTestBase with Seri
     util.verifyExplain(stmt, ExplainDetail.JSON_EXECUTION_PLAN)
   }
 
+  @Test
+  def testJoinWithLookupJoinHint(): Unit = {
```

Comment:
this can be merge with the 'testJoinWithInvalidPropagationToSubQuery', rename to 'testJoinHintWithNoPropagatingToSubQuery'